submissions/cynthiaNwakaeme/CSS/contact.css

```diff
@@ -0,0 +1,277 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 15px;
+    padding: 0;
+    margin: 0;  
+}
+.nav-container{
+    width: 95%;
+    margin: auto;
+    overflow: hidden;
+}
+ul{ 
+    padding: 0;
+    margin: 0;
+}
+/* THE HEADER */
+header{
+    background:#9d426b; 
+    color: white;
+    padding-top: 10px;
+    min-height: 15%;
+    border-bottom: goldenrod 3px solid;
+    
+}
+header a{
+    color: white;
+    text-decoration: none;
+    font-size: 16px;
+}
+
+header li{
+    display: inline;
+    padding:0 30px;
+}
+header #nav-branding{
+    float: left;
+    padding-bottom: 10px;
+}
+header #nav-branding h2,header #nav-branding p{
+    margin:0;
+    padding:0;
+    color:white;
+}
+header nav{
+    float: right;
+    margin-top: 15px;
+}
+header a:hover{
+    text-transform: uppercase;
+    color:white;
+    font-weight: bold;
+
+}
+  
+/* THE CONTACT INFORMATION */
+.container{
+    overflow: hidden;
+    padding: 30px 0;
+    margin: auto;
+    width: 90%;
+}
+.container img{
+    width: 35px;
+    text-align: center;
+    word-spacing: 2px;
+    height: 35px;
+}
+        /* MAIN FRAME */
+main{
+    float: left;
+    text-align: center;
+    width: 50%;
+    padding-top: 3%;
+    
+}
+
+main h3, aside h3{
+    color: rgb(58, 57, 57);
+}
+main ul li{
+    display: inline;
+    padding: 2%;
+}
+
+       /* ASIDE FRAME */
+aside{
+    float: left;
+    width: 43%;
+    margin-left: 5%;
+}
+.enquiry-details{
+    overflow: hidden;
+    padding: 20px;
+    width: 90%;
+    border:1px solid rgb(129, 129, 129);
+    border-radius: 3%;
+    margin-bottom: 10px;
+}
+.enquiry-details img{
+    padding: 0 45%;
+}
+.enquiry-details h3{
+    text-align: center;
+    width: 95%; 
+}
+.enquiry-details input{
+    border-top-style: hidden;
+    border-right-style: hidden;
+    border-left-style: hidden;
+    border-bottom: 1px groove rgb(129, 129, 129);
+    width: 95%;
+    padding-top: 5%;
+}
+.enquiry-details input:focus{
+    outline: none;
+
+  }
+.enquiry-details input::placeholder{
+    color:rgb(129, 129, 129) ;
+    font-size: 15px;
+}
+.enquiry-details label{
+    padding-top: 5%;
+    color:rgb(129, 129, 129) ;
+}
+.enquiry-details textarea{
+    resize: none;
+    width: 95%; 
+}
+.enquiry-details button{
+    float: right;
+    width: 25%;
+    height: 40px;
+    background-color: #9d426b;
+    color: #fff;
+    margin: 2% 4% 0;
+    padding: 2%;
+    border: none;
+    font-size: 18px;
+}
+
+/* THE FOOTER */
+footer{
+    background-color: #9d426b;
+    padding:20px 0 0 50px;
+    color: white;
+    overflow: hidden;
+    clear: both;
+}
+.footer-box,
+.footer-box-2,
+.footer-box-3{
+    float: left;
+    width: 33%;
+}
+.footer-box ul li,
+.footer-box-3 ul li
+{
+    list-style: none;
+    margin-bottom: 8px;
+}
+.footer-box ul li a{
+    text-decoration: none;
```

Comment:
You can do this ` a {text-decoration: none;}` for all the links instead of declaring it one after the other.

Reply:
@Cynthia-Berry Where did you declare it. I can still see it been other declarations.

Reply:
What i did was to do a generic declaration of `li{text-decoration:none}` However i had issues for `li` tags with `a` tags embedded inside, hence i had to trigger the parent `li` tags then style `a` tags to be able to effect my changes.

@ToluAkin ,The snipped files i have are resent snipped files. I took off the cart icon i did using HTML element.
You can also confirm my claims from there.

I am sorry about the inconvinences.
Thank you ma.